config/install/standard/kuadrant-version.yaml

```diff
@@ -3,4 +3,4 @@ kind: CatalogSource
 metadata:
   name: kuadrant-operator-catalog
 spec:
-  image: quay.io/kuadrant/kuadrant-operator-catalog:v1.0.0-rc6 #change this to the version you want to install
+  image: quay.io/kuadrant/kuadrant-operator-catalog:latest #change this to the version you want to install
```

Comment:
This is being changed to "latest" because it's on main, this will now get updated to the correct image tag as part of the release process see https://github.com/Kuadrant/kuadrant-operator/pull/1053/files#r1860933307

Installing different versions should be done by checking out the correct version before applying:
```
git checkout v1.0.0-rc10
kubectl apply -k config/install/standard
```
or, ideally using a remote ref:

```
kubectl apply -k github.com/kuadrant/kuadrant-operator/config/install/standard?ref=v1.0.0-rc10
```

Reply:
I don't think the remote will work as the env files are not present

Reply:
I'm probably going to close this PR in favour of using kube burner for building the same scale testing scenarios. That said, this image ref on main should probably be pointing to something other than a fixed tag that never gets updated as part of the release. If you are referring to the example apply command i.e. `kubectl apply -k github.com/kuadrant/kuadrant-operator/config/install/standard?ref=v1.0.0-rc10`, it was just an example of how it could be used if you started updating this image correctly as part of releases.